src/query/sql/src/planner/dataframe.rs

```diff
@@ -312,18 +321,20 @@ impl Dataframe {
 
     pub async fn sort(
         mut self,
-        select_list: Vec<Expr>,
-        order_by: &[OrderByExpr],
+        columns: &[&str],
+        order_by: Vec<(Expr, Option<bool>, Option<bool>)>,
```

Comment:
It's OK to omit `columns`?

Reply:
Good question. add sort_column and distinct_column

https://github.com/datafuselabs/databend/pull/12298/commits/b39cebaf1e4641d0909621d3bf5c6967abe2a529